CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ This release has an [MSRV][] of 1.75.
 
 ### Changed
 
+- Updated `wgpu` to 22.1.0. ([#635] by [@waywardmonkeys])
```

Comment:
We don't have a consistent policy here. Is it worth marking this as a breaking change?

Reply:
I think it should be for now ... if we don't, it would cause some problems for users of `bevy_vello`.

Reply:
I wasn't suggesting we don't treat it as a breaking change. I mean to ask whether we should call it out as a breaking change in the changelog

Reply:
Oh, sorry. Doing too many things at once. How should we do that? I'm okay with doing so here.

Reply:
I did this like your own entry now. Will merge soon.